discuss/email_servers.rst

```diff
@@ -35,6 +34,13 @@ Here is a typical configuration for a G Suite server.
 
 Then set your email domain name in the General Settings.
 
+.. note:: If you get a *[AUTHENTICATIONFAILED] Invalid credentials (Failure)* warning when you *Test
+   Connection* on a Gmail address, activate the *Less secure app access* option.
+   A direct link can be `accessed here. <https://myaccount.google.com/lesssecureapps?pli=1>`_
```

Comment:
Super nitpicking: Move the punctuation point after the url so that it's not part of the hyperlink.